gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-account/src/main/java/io/gravitee/am/gateway/handler/account/resources/account/AccountResponseHandler.java

```diff
@@ -70,4 +85,32 @@ private static String getUpdateUserResponseJson() {
     private static String getTemporaryWipResponseJson() {
         return new JsonObject().put("temp", "true").put("reason", "wip").toString();
     }
+
+    public static void handleNoBodyResponse(RoutingContext routingContext) {
+        routingContext.response().setStatusCode(204).end();
+    }
+
+    public static Handler<RoutingContext> handleEnrollNeedChallenge(RoutingContext routingContext, Factor factor, User user) {
```

Comment:
Do we need `RoutingContext routingContext` since the handler has already a `routingContext1` ?

Reply:
for the update user factor method, to remain generic enough for reusability I need to return the handler, and when defining the handler as a lamba reference it's kinda interfering in scope causing this naming confict, however i need the routingctx argument